perfkitbenchmarker/linux_packages/ycsb.py

```diff
@@ -627,8 +627,11 @@ def _Run(self, vm, **kwargs):
       param, value = pv.split('=', 1)
       kwargs[param] = value
     command = self._BuildCommand('run', **kwargs)
-    stdout, _ = vm.RobustRemoteCommand(command)
-    return ParseResults(str(stdout))
+    # YCSB version greater than 0.7.0 output some of the
```

Comment:
Yes, we still need that, since stderr contains not only the messages we need but also a lot of debugging, error messages.